From b63b995b1691c0448fb14bcf23341651c1f0679c Mon Sep 17 00:00:00 2001
From: Lokesh Vutla <lokeshvutla@ti.com>
Date: Mon, 21 Aug 2017 12:51:00 +0530
Subject: [PATCH 046/256] arm: dts: Add u-boot specific compatibles

Separate out u-boot specific compatibles from dts files.
This will help in syncing dts files in future.
Also these will get deleted eventually once respective drivers
are capable of handling Linux dts files.

Signed-off-by: Lokesh Vutla <lokeshvutla@ti.com>
Reviewed-by: Tom Rini <trini@konsulko.com>
---
 arch/arm/dts/dra71-evm-u-boot.dtsi      | 23 +++++++++++++++++++++++
 arch/arm/dts/dra72-evm-revc-u-boot.dtsi | 23 +++++++++++++++++++++++
 arch/arm/dts/dra76-evm-u-boot.dtsi      | 15 +++++++++++++++
 arch/arm/dts/omap5-u-boot.dtsi          |  7 +++++++
 4 files changed, 68 insertions(+)
 create mode 100644 arch/arm/dts/dra71-evm-u-boot.dtsi
 create mode 100644 arch/arm/dts/dra72-evm-revc-u-boot.dtsi
 create mode 100644 arch/arm/dts/dra76-evm-u-boot.dtsi

diff --git a/arch/arm/dts/dra71-evm-u-boot.dtsi b/arch/arm/dts/dra71-evm-u-boot.dtsi
new file mode 100644
index 0000000000..8ae64c0a3c
--- /dev/null
+++ b/arch/arm/dts/dra71-evm-u-boot.dtsi
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include "omap5-u-boot.dtsi"
+
+&pcf_gpio_21{
+	u-boot,i2c-offset-len = <0>;
+};
+
+&pcf_hdmi{
+	u-boot,i2c-offset-len = <0>;
+};
+
+&cpsw_emac0 {
+	phy-handle = <&dp83867_0>;
+};
+
+&cpsw_emac1 {
+	phy-handle = <&dp83867_1>;
+};
diff --git a/arch/arm/dts/dra72-evm-revc-u-boot.dtsi b/arch/arm/dts/dra72-evm-revc-u-boot.dtsi
new file mode 100644
index 0000000000..8ae64c0a3c
--- /dev/null
+++ b/arch/arm/dts/dra72-evm-revc-u-boot.dtsi
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include "omap5-u-boot.dtsi"
+
+&pcf_gpio_21{
+	u-boot,i2c-offset-len = <0>;
+};
+
+&pcf_hdmi{
+	u-boot,i2c-offset-len = <0>;
+};
+
+&cpsw_emac0 {
+	phy-handle = <&dp83867_0>;
+};
+
+&cpsw_emac1 {
+	phy-handle = <&dp83867_1>;
+};
diff --git a/arch/arm/dts/dra76-evm-u-boot.dtsi b/arch/arm/dts/dra76-evm-u-boot.dtsi
new file mode 100644
index 0000000000..b007f78d46
--- /dev/null
+++ b/arch/arm/dts/dra76-evm-u-boot.dtsi
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) 2017 Texas Instruments Incorporated - http://www.ti.com/
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ */
+
+#include "omap5-u-boot.dtsi"
+
+&cpsw_emac0 {
+	phy-handle = <&dp83867_0>;
+};
+
+&cpsw_emac1 {
+	phy-handle = <&dp83867_1>;
+};
diff --git a/arch/arm/dts/omap5-u-boot.dtsi b/arch/arm/dts/omap5-u-boot.dtsi
index 17602909a6..2eeed6f4a0 100644
--- a/arch/arm/dts/omap5-u-boot.dtsi
+++ b/arch/arm/dts/omap5-u-boot.dtsi
@@ -8,6 +8,10 @@
  */
 
 /{
+	chosen {
+		tick-timer = &timer2;
+	};
+
 	ocp {
 		u-boot,dm-spl;
 
@@ -19,10 +23,12 @@
 
 &uart1 {
 	u-boot,dm-spl;
+	reg-shift = <2>;
 };
 
 &uart3 {
 	u-boot,dm-spl;
+	reg-shift = <2>;
 };
 
 &mmc1 {
@@ -49,6 +55,7 @@
 	u-boot,dm-spl;
 
 	m25p80@0 {
+		compatible = "spi-flash";
 		u-boot,dm-spl;
 	};
 };
-- 
2.13.6

